src/realm/list.cpp

```diff
@@ -371,6 +371,16 @@ Obj LnkLst::create_and_insert_linked_object(size_t ndx)
 Obj LnkLst::create_and_set_linked_object(size_t ndx)
 {
     Table& t = *get_target_table();
+    if (t.is_embedded() && m_list.get(ndx) != ObjKey{}) {
```

Comment:
When would `m_list.get(ndx)` give `ObjKey{}` here?

Reply:
This got removed.